### PR TITLE
Enable TOML reformatting in prettier

### PR DESCRIPTION
**Pull request by ssbarnea**
- enable TOML reformatting in prettier

##### Contributor checklist

- [x] Provided the tests for the changes.
- [x] Assure PR title is short, clear, and good to be included in the user-oriented changelog

##### Maintainer checklist

- [x] Assure one of these labels is present: `backwards incompatible`, `feature`, `enhancement`, `deprecation`, `bug`, `dependency`, `docs` or `skip-changelog` as they determine changelog listing.
- [x] Assign the PR to an existing or new milestone for the target version (following [Semantic Versioning](https://blog.versioneye.com/2014/01/16/semantic-versioning/)).

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -36,9 +36,14 @@ repos:
     rev: 1.7.4
     hooks:
       - id: bandit
-        args: [--ini, .bandit]
+        args: [-c, pyproject.toml]
```
Comment (atugushev) [inline .pre-commit-config.yaml:39]: I wonder what's your opinion on 3rd party configs on: having all in `pyproject.toml` or separate files (e.g. `.flake8`, `.coveragerc`, etc)?

Comment (webknjaz) [reply .pre-commit-config.yaml:39]: It's best to have separate configs because it is hard to maintain a combined mess in a single one.

Comment (ssbarnea) [reply .pre-commit-config.yaml:39]: If tool authors would have considered that "combined mess" is a real problem, they would have prevented enablement of that feature, but [almost all adopted it](https://github.com/orgs/stdize/discussions/1).

I (personally) find the benefit of decluttering the project root folder as adding more value than the downside of keeping everything in one file, especially as these settings do not need lots of updates and we already have a formatter for it. In fact I suspect that passing the name of the config file in pre-commit config is not even needed.

In general TOML is far more safe file format than old INIs, so the change of bugs with them is considerably lower. I am sure that bugs about inline comments and multiline entries are not unknown to you.

Comment (atugushev) [reply .pre-commit-config.yaml:39]: I agree on `ini`-mess, however long `pyproject.toml` with ombined everything can become huge if we adopt more and more tools that require some configuration. This is just my personal opinion based on a habit to organise code namespaces/modules as much as possible. 

What if we create a `.bandit.toml` file and  pass it `args: [-c, .bandit.toml]` to get rid of ini config at least fo bandit?

Comment (ssbarnea) [reply .pre-commit-config.yaml:39]: Bad idea, unless that filename in particular is documented as recognized by the tool.

Comment (ssbarnea) [reply .pre-commit-config.yaml]: I removed bandit change for PR due to current controversy.

**Review by atugushev — APPROVED** [review]

**Pull request merged**